docs/how-to-open-a-pull-request.md

```diff
@@ -37,13 +37,13 @@ An example is `fix(learn): Fixed tests for the do...while loop challenge`.
 
     - You will be presented with a pull request template. This is a checklist that you should have followed before opening the pull request.
 
-    - Fill in the details as they seem fit you. This information will be reviewed and decide whether or not, your pull request is going to be accepted.
+    - Fill in the details as you see fit. This information will be reviewed, and the reviewers will decide whether or not your pull request is accepted.
```

Comment:
```suggestion
    - Fill in the details as you see fit. This information will be reviewed and reviewers will decide whether or not your pull request is accepted.
```

Reply:
This is not my PR, and I cannot commit this suggestion via the interface. I made the original suggested changes on behalf of the OP in hopes to get this PR merged and I think it's fine how it is. If you want it this way, you should make the changes.

Reply:
I am going to add a commit.